pkg/manifest/os_test.go

```diff
@@ -369,3 +370,63 @@ func TestHostnameDoesNotIncludeHostnameStage(t *testing.T) {
 	st := manifest.FindStage("org.osbuild.hostname", pipeline.Stages)
 	require.Nil(t, st)
 }
+
+func TestAddFilesOS(t *testing.T) {
+	os := manifest.NewTestOS()
+
+	require := require.New(t)
+
+	// add some files to the OSCustomizations which are included near the end
+	// of the pipeline
+	fileOne, err := fsnode.NewFile("/etc/test/one", nil, nil, nil, []byte("test 1"))
```

Comment:
(super nitpick) this could be `fileOne := common.Must(fsnode.NewFile(...` but not sure if that would improve readability much

Reply:
Might be nicer to  get an error from `require` than a `panic()`, but `fsnode.NewFile()` shouldn't return an error unless the path is bad (and the user and group have bad types), so it's probably fine to use `Must()`.

Reply:
Done.